component/site/views/default/helpers/defaultloadedfromtemplate.php

```diff
@@ -1247,6 +1247,8 @@ function DefaultLoadedFromTemplate($view, $template_name, $event, $mask, $templa
 		$template_value = $tmprow->text;
 		$template_value = str_replace("@£@", "@", $template_value);
 
+		//We add new line characters again to avoid being marked as SPAM when using tempalte in emails
+		$template_value = preg_replace("@(<\s*(br)*\s*\/\s*(p|td|tr|table|div|ul|li|ol|dd|dl|dt)*?\s*>)+?@i","$1\n",$template_value);
```

Comment:
Actually the *dt)&#42;?* is irrelevant as it will only be useful in tags like:
```html
</pp>
```
For the rest:
* *<\s&#42;*  0 or any white spaces after the <
* *(br)&#42;* 0 or any BR
*  *\s&#42;* 0 or any white spaces
*  *)+?* Will apply to 1 or any patterns that matches the group

I have applied a couple of extra changes that better match. Please consider following gist I created for testing in case you want to do further testS:

https://gist.github.com/carcam/83456db4ec495dccafb9